app/api/events.py

```diff
@@ -132,6 +133,12 @@ def after_create_object(self, event, data, view_kwargs):
                                  status='accepted')
         save_to_db(role_invite, 'Organiser Role Invite Added')
         if data.get('original_image_url'):
+            try:
+                image_file = urllib.request.urlopen(data['original_image_url']).read()
+            except (urllib.error.HTTPError, urllib.error.URLError):
+                raise UnprocessableEntity(
+                    {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
+                )
```

Comment:
This just moved the util method code inline. How does this fix the issue? This is even worse than before

This even removed the comment containing important discussion as to what was to be done.

Please iterate what your follow up commit would be before sending it so that everyone is on the same page